packages/app/app/components/ArtistView/ArtistLink/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Button } from 'semantic-ui-react';
+import FontAwesome from 'react-fontawesome';
+
+const shell = require('electron').shell;
+
+const ArtistLink = ({ artistLink, size, label='Visit', circular=false}) => (
+  <Button
+    primary
+    onClick={() => shell.openExternal(artistLink)}
```

Comment:
Can you put this in `useCallback`?

Reply:
@nukeop Do you mean Memoize the functional component and usecallback for onclick handler right ? Because Memoize is usually used for efficient rendering.Do correct me if I am wrong.

Reply:
No need to memoize the whole component, just put the onclick handler in a `useCallback` hook so we don't re-create this function on each render.